main.tf

```
@@ -5,14 +5,15 @@ locals {
 }
 
 module "label" {
-  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.14.1"
-  enabled    = var.enabled
-  namespace  = var.namespace
-  stage      = var.stage
-  name       = var.name
-  delimiter  = var.delimiter
-  attributes = var.attributes
-  tags       = var.tags
+  source              = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.14.1"
```

```suggestion
  source              = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.16.0"
```

let's update to the latest version

---

What do you think of merging this one first: https://github.com/cloudposse/terraform-aws-ecr/pull/35 and then rebasing this PR against the master. In that case, this particular PR will stay cleaner introducing only one feature.

---

agree
testing #35  now and will merge if ok